pkg/agent/openflow/multicluster.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+import (
+	"net"
+
+	"antrea.io/antrea/pkg/agent/openflow/cookie"
+	binding "antrea.io/antrea/pkg/ovs/openflow"
+)
+
+var GlobalVirtualMACForMulticluster, _ = net.ParseMAC("ff:ee:dd:cc:bb:aa")
```

Comment:
There aren't many such special MACs if this just reverses the existing one, maybe let's use "aa:bb:cc:dd:ee:f0" for this case so we can say "aa:bb:cc:dd:ee:fX" is antrea-specific virtual MACs.

Reply:
Done, update it to `aa:bb:cc:dd:ee:f0`